winrm.go

```diff
@@ -44,22 +46,42 @@ func main() {
 	flag.BoolVar(&https, "https", false, "use https")
 	flag.BoolVar(&insecure, "insecure", false, "skip SSL validation")
 	flag.StringVar(&cacert, "cacert", "", "CA certificate to use")
+	flag.StringVar(&cert, "cert", "", "Cert")
+	flag.StringVar(&key, "key", "", "Key")
 	flag.Parse()
 
-	var certBytes []byte
 	var err error
-	if cacert != "" {
-		certBytes, err = ioutil.ReadFile(cacert)
+	var certBytes, keyBytes []byte
+	if cert != "" && key != "" {
+		certBytes, err = ioutil.ReadFile(cert)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		keyBytes, err = ioutil.ReadFile(key)
 		if err != nil {
 			fmt.Println(err)
 			os.Exit(1)
 		}
 	} else {
 		certBytes = nil
+		keyBytes = nil
+	}
+
+	var CAcertBytes []byte
```

Comment:
Can you do some research on whether the CACertBytes is related to the insecure flag at all? As in, when insecure is enabled it's needed and when it's disable it's not needed and if it is so can you make sure that's enforced? 

I know it's not related to this PR's main content, but it's a nice drive-by fix in case it works and you're willing to do it